### PR TITLE
chore: add contributing doc

### PR DESCRIPTION
**Pull request by westeezy**
borrowing the [CONTRIBUTING.md](https://github.com/paypal/paypal-checkout-components/blob/4feda2dc411698f093dc0aecb5f384b580f49a2e/CONTRIBUTING.md) from checkout-components to have one in the repo. We can expand this overtime.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+1. Ensure that no errors are generated by ESLint (run during `$ npm test`).
+2. Commit your changes in logical chunks, i.e. keep your changes small per single commit.
+3. Locally merge (or rebase) the upstream branch into your topic branch: `$ git pull upstream && git merge`.
+4. Push your topic branch up to your fork: `$ git push origin <topic-branch-name>`.
+5. Open a [Pull Request](https://help.github.com/articles/using-pull-requests) with a clear title and description.
+6. You should follow the [Angular.js commit message guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) for both your commit messages and PR titles.
+
+If you have any questions about contributing, please feel free to contact us by posting your questions on GitHub.
+
+Copyright 2020-present, PayPal under [the Apache 2.0 license](LICENSE.txt).
```
Comment (gregjopa) [inline CONTRIBUTING.md:48]:
```suggestion
Copyright 2020-present, PayPal, Inc. under [the Apache 2.0 license](LICENSE.txt).
```

I also think we are techically PayPal, Inc. per the privacy legal thing no one reads: https://www.paypal.com/us/webapps/mpp/ua/privacy-full

**Review by gregjopa — APPROVED** [review]
💯

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+$ git remote add upstream git@github.com:paypal/react-paypal-js.git
+
+# Get the latest upstream changes
+$ git pull upstream
+
+# Install dependencies
+$ npm install
+
+# Run scripts to verify installation (Note: test includes lint)
+$ npm test
```
[the remark this answers is not in the document]
Comment (gregjopa) [reply CONTRIBUTING.md:29]: The nice thing with `npm run validate` is it also makes sure the build process works.

[the remark this answers is not in the document]
Comment (westeezy) [reply CONTRIBUTING.md:29]: making it validate 🙌

**Pull request merged**